Doc/whatsnew/3.9.rst

```diff
@@ -626,6 +626,10 @@ Deprecated
   `parso`_.
   (Contributed by Carl Meyer in :issue:`40360`.)
 
+* The :mod:`xml.etree.cElementTree` module has been deprecated since Python
+  3.3 and is scheduled to be removed in Python 3.10.
```

Comment:
Use more direct wording:
* clearer
* similar to that in xml.etree.elementtree.rst
* fewer words = more readable

```suggestion
  3.3 and will be removed in Python 3.10.
```